style.css

```
@@ -0,0 +1,494 @@
+*{
+    margin: 0 auto;
+    padding: 0;
+
+    box-sizing: border-box;
+}
+body{
+    position: relative;
+
+    width: 100vw;
+    height: 100vh;
+    
+}
+header{
+    background:url(./images/t.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-blend-mode: multiply;
+
+    display: flex;
+
+    width: 100vw;
+    height: 70px;
+
+}
+.brand{
+    position: absolute;
+    width: 100%;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.brand-name{
+    font-family: 'Caramel', cursive;
+    font-size: 42px;
+    color: palevioletred;
+
+    animation: web 5s linear infinite;
+}
+.discription{
+    font-family: 'Covered By Your Grace', cursive;
+    font-size: 15px;
+}
+/*toggle sidebar*/
+.toggle{
+    position: relative;
+
+    width: 100%;
+    height: 80px;
+}
+.sidebar{
+    position: fixed;
+
+    width: 250px;
+    height: 100vh;
+    right: 0;
+    top: 0;
+    margin-top: 70px;
+
+    transform: translateX(250px);
+
+    transition: transform 250ms ease-in-out;
+
+    font-family: cursive, serif;
+    background: #1f0003b7;
```

Try this and let me know how it looks. You can also use background: transparent; on this
```suggestion
    background: #1f0003b7; (background: transparent)
    backdrop-filter: blur(14px); (set the px according to your preferance)
```

---

yup its amazing this looks greats i notice that blurry effect